src/main/java/bowling/domain/frame/FinalFrame.java

```diff
@@ -0,0 +1,79 @@
+package bowling.domain.frame;
+
+import bowling.domain.dto.FrameResponse;
+import bowling.domain.state.PitchState;
+
+public class FinalFrame implements Frame {
+    private Round round;
+    private Frame frame;
```

Comment:
frame이 이전 frame으로 보이는데 맞을까요? 😄 좀 더 의미를 알기 쉬운 이름으로 바뀐다면 좋겠습니다.
```suggestion
    private Frame previousFrame;
```